### ISSUE TITLE
Bug with messageInput

### ISSUE THREAD

**Issue by agdemore** [markussammallahti/harbour-slackfish#7]
When you send message the text in TextArea is not cleared.

I found some weird bug in Silica.
Before you clear text in input like `messageInput.text = ""` you must add space after text like this:
```
messageInput.text = messageInput.text + " ";
messageInput.text = ""
```
MessageInput.qml, line: 63

P.S. 
Sorry I did not make a pull request, I have a lot of work

**Comment by markussammallahti** [comment]
Thanks for the info, I've been wondering why that sometimes happens. I'll fix this in next release.

**Comment by markussammallahti** [comment]
Fixed in d0f4d474876f0631c7eb7201082fecf990565423 and will be included in next release.
